lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -70,7 +70,7 @@ def export_kymotrackgroup_to_csv(
 
     Parameters
     ----------
-    filename : str
+    filename : str | os.PathLike
```

Comment:
Elsewhere, we've always been writing `or` in full rather than `|`.

Reply:
Over the last few years, a lot of work has been put into standardizing the Python type hinting. Various docstring formats used to have their own conventions but it's nice to adopt the new standard way. There's a lot of good info and links here (note the 3 links near the top, including the cheatsheet): https://docs.python.org/3/library/typing.html

Specifically for this case, `Union[A, B, C]` was initially set up, but it has since been refined to the `A | B | C` syntax which is pretty nice. Similarly, `Optional[T]` became `T | None`.